python/pyspark/ml/tests.py

```diff
@@ -684,12 +684,32 @@ def test_logistic_regression(self):
         except OSError:
             pass
 
+    def _compare_params(self, m1, m2, param):
+        """
+        Compare 2 ML params, assert they have the same param. The param must be a parameter of m1.
```

Comment:
"Compare 2 ML params, assert they have the same param."
--> "Compare 2 ML Params instances for the given param, and assert both have the same param value and parent."